wpa_gui.desktop: add keywords entry

According to the XDG, desktop files should have a Keywords entry, see:
http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s05.html

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/wpa_supplicant/wpa_gui-qt4/wpa_gui.desktop
+++ b/wpa_supplicant/wpa_gui-qt4/wpa_gui.desktop
@@ -8,3 +8,4 @@ GenericName=wpa_supplicant user interfac
 Terminal=false
 Type=Application
 Categories=Qt;Network;
+Keywords=wlan;wifi;configuration;
